feedback.html

```diff
@@ -0,0 +1,169 @@
+---
+layout: page
+order: 0
+section: "2018"
+title: "Feedback: How’d we do?"
+titleDisplay: "Feedback"
+redirect_from:
+  - "/2018/feedback"
+---
+
+Help us make next year better! We value your feedback as we work to improve this event. Survey responses are anonymous and no questions are required (we use formspree.io to send responses).
+
+Thank you &lt;3
+
+<hr />
+
+<form action="https://formspree.io/orga@ournetworks.ca" method="POST">
+  <div class="row form-group">
+    <div class="ten columns">
+      <fieldset id="recommend">
+        <legend>How likely is it that you'd recommend Our Networks to a friend or family member?</legend>
+        <div class="form-radio-range">
+          {% include forms/form-radio-scale.html name="recommend" start=0 end=10 %}
+          <div class="form-radio-range-label">
+            <div class="range-left">Not likely at all</div>
+            <div class="range-right">Extremely likely</div>
+          </div>
+        </div>
+
+      </fieldset>
+    </div>
+  </div>
+  <div class="row form-group">
+    <div class="ten columns">
+      <label for="good">What was Our Networks good at?</label>
+      <input type="text" id="good" class="input-text" name="good">
+    </div>
+  </div>
+  <div class="row form-group">
+    <div class="ten columns">
+      <label for="do-better">What could Our Networks do better?</label>
+      <input type="text" id="do-better" class="input-text" name="do-better">
+    </div>
+  </div>
+  <div class="row form-group">
+    <div class="ten columns">
+      <fieldset id="expectations">
+        <legend>Overall, how well did Our Networks meet your expectations?</legend>
+        <div class="form-radio-range six columns">
+          {% include forms/form-radio-scale.html name="expectations" start=1 end=5 %}
+          <div class="form-radio-range-label">
+            <div class="range-left">Not well at all</div>
+            <div class="range-right">Extremely well</div>
+          </div>
+        </div>
+
+      </fieldset>
+    </div>
+  </div>
+  <div class="row form-group">
+    <div class="ten columns">
+      <label for="reason">What best describes your reason for attending Our Networks 2018?</label>
+      <input list="reasons" id="reason" name="reason">
+      <datalist id="reasons">
+        <option value="The topic interested me">
+        <option value="I wanted to contribute my skills to to a meaningful effort">
+        <option value="I wanted to connect with others who care about the same issues that I do">
```

Comment:
I think we should cut "that I do"... this continues to crop wierd when I test on different displays